src/item-navigator.ts

```diff
@@ -181,44 +209,24 @@ export class ItemNavigator
     ></ia-no-theater-available>`;
   }
 
-  get theaterSlot() {
-    return html`
-      <slot name="theater-main" style=${this.readerHeightStyle}></slot>
-    `;
-  }
-
-  get booksViewer(): TemplateResult {
-    const slotVisibility = !this.loaded ? 'opacity: 0;' : 'opacity: 1;';
-
-    return html`
-      <book-navigator
-        .modal=${this.modal}
-        .baseHost=${this.baseHost}
-        .itemMD=${this.item}
-        ?signedIn=${this.signedIn}
-        ?sideMenuOpen=${this.menuOpened}
-        .sharedObserver=${this.sharedObserver}
-        @ViewportInFullScreen=${this.manageViewportFullscreen}
-        @loadingStateUpdated=${this.loadingStateUpdated}
-        @updateSideMenu=${this.manageSideMenuEvents}
-        @menuUpdated=${this.setMenuContents}
-        @menuShortcutsUpdated=${this.setMenuShortcuts}
-      >
-        <div slot="theater-main" style=${slotVisibility}>
-          ${this.theaterSlot}
-        </div>
-      </book-navigator>
-    `;
-  }
-
   get renderViewport(): TemplateResult | typeof nothing {
     if (!this.item) {
       return nothing;
     }
-    if (this.itemType === ItemType.BOOK) {
-      return this.booksViewer;
+    if (this.itemType !== ItemType.OPEN) {
```

Comment:
question: can this be anything but `ItemType.OPEN` right now? Won't anything else get a type error because there's only one enum?

Reply:
no longer used. thanks again for the push!